app/views/application/_show_page.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="row">
+  <div class="col-lg-12">
+    <h1><%= item.name %> <small><%= create_verb %> by: <%= created_by %></small></h1>
+    <h4>Ranked: <%= item.ranking %></h4>
+    <p><%= item.description %></p>
+      <%= button_to "Upvote", polymorphic_path([:upvote, item]), class: "btn btn-primary" %>
+      <%= link_to "Edit #{item.name}", edit_polymorphic_path(item, id: params[:id]), class: "btn btn-default" %>
+      <%= link_to "DELETE", polymorphic_path(item, id: params[:id]), method: :delete, class: "btn btn-danger",
+      data: { confirm: "Are you sure?"} %>
+      <%= link_to "View All #{name.capitalize}", polymorphic_path(name), class: "btn btn-default" %>
+      <%= link_to "View All Media", "/", class: "btn btn-default" %>
+  </div>
+</div>
```

Comment:
Great, that you used also partials for index and show page, it's really easy to read your code.